scripts/packer.sh

```diff
@@ -53,13 +53,18 @@ packer_in_container() {
       CDP_TELEMETRY_VERSION=$(curl -L -k -s ${CDP_TELEMETRY_BASE_URL}AVAILABLE_VERSIONS | head -1)
     fi
     CDP_TELEMETRY_RPM_URL="${CDP_TELEMETRY_BASE_URL}cdp_telemetry-${CDP_TELEMETRY_VERSION}.x86_64.rpm"
+    CDP_TELEMETRY_RPM_URL="https://cloudera-build-us-west-1.vpc.cloudera.com/s3/build/39697508/cdp-infra-tools/1.x/redhat8/yum/cdp_telemetry-0.4.36.x86_64.rpm"
```

Comment:
If we hardcode the rpm url here, the logic above does not make sense.

Reply:
I assume this is a temporary override, right? If so, please clean this up.

Reply:
Extended with comment.